docs/en/writing-running-appium/tutorial/swipe-tutorial.md

```diff
@@ -0,0 +1,44 @@
+## Scroll / Swipe actions tutorial
+
+### Android vs iOS UI differences
+
+Appium uses XCUITest driver for iOS and UIAutomator2, Espresso for
+Android. The default driver for Android is UIAutomator2.
+
+With default driver settings we can see elements outside of the view
```

Comment:
I'd like to leave a short comment about such limitation is basically the root framework, XCTest by Apple and Android by Google.